DiscordChatExporter.Cli/Commands/ExportChannelCommand.cs

```diff
@@ -12,7 +12,10 @@ public class ExportChannelCommand : ExportCommandBase
             Description = "Channel ID.")]
         public string ChannelId { get; set; } = "";
 
-        public override async ValueTask ExecuteAsync(IConsole console) =>
+        public override async ValueTask ExecuteAsync(IConsole console)
+        {
+            base.ExecuteAsync();
```

Comment:
Missing `await` here.

Reply:
Also, is this the only place which needs to be updated? Are there any others where we need to call `base.ExecuteAsync()`?